content/api/configuration.md

```diff
@@ -1,10 +1,1407 @@
 ---
 title: Configuration
 ---
-> configuration file
-> possible extensions, i. e. .babel.js
-> exporting a function and --env
-> returning a Promise
-> exporting multiple configurations
 
-> see also [[Using the Cli]]
+?> TODO: Add a reference to [[Using the Cli]]
+
+### Passing a Configuration
+
+?> TODO: exporting a function and --env
+
+?> TODO: returning a Promise
+
+?> TODO: exporting multiple configurations
+
+---
+
+### External Configurations
+
+?> TODO: possible extensions, i. e. .babel.js
+
+---
+
+Webpack is fed a configuration object. It is passed in one of two ways depending on how you are using webpack: through the Command Line (CLI) or via Node. Either method also allows an array of configuration objects, which are processed in parallel. This is more efficient than calling webpack multiple times. All the available configuration options are specified below...
```

Comment:
Command Line (CLI) -> Command Line Interface (CLI). Or go with terminal since that's well known word for it too.

I would drop the `...` from the end as it leaves the sentence hanging.